.github/workflows/tests-pull-request.yaml

```diff
@@ -109,13 +109,15 @@ jobs:
             ${{ runner.os }}-ruby
       - name: Run fixtures.
         run: psql -h 127.0.0.1 -p 5432 -U postgres -f ./tests/fixture.sql
+      - name: Prepare packages.
+        run: |
+          sudo apt-get update &&  sudo apt-get install ruby-bundler ruby-dev -y &&
```

Comment:
[`cache-apt-pkgs-action`][1] could be used here to improve CI run time

[1]: https://github.com/marketplace/actions/cache-apt-packages